.github/workflows/release.yml

```diff
@@ -192,24 +192,57 @@ jobs:
           files: |
             ./tmp/${{ env.artifact_name }}.tgz
 
-#  update-homebrew-formula:
-#    name: Run brew bump-formula-pr for iota on testnet releases
-#    needs: release-build
-#    runs-on: ubuntu-latest
-#    # releasing iota cli on testnet releases because it lags `main` less than mainnet, but is more likely to be stable than devnet
-#    if: ${{ github.event_name == 'release' && contains( inputs.iota_ref, '-rc') }}
-#    steps:
-#      - uses: mislav/bump-homebrew-formula-action@b3327118b2153c82da63fd9cbf58942146ee99f0 # v3.1.0
-#        with:
-#          formula-name: iota
-#          create-pullrequest: true
-#          tag-name: "${{ env.iota_ref }}"
-#          commit-message: |
-#            {{formulaName}} ${{ env.iota_ref }}
-#
-#            Created by https://github.com/mislav/bump-homebrew-formula-action
-#
-#            From release: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
-#        env:
-#          # https://github.com/settings/tokens/new?scopes=public_repo,workflow
-#          COMMITTER_TOKEN: ${{ secrets.HOMEBREW_GH_FORMULA_BUMP }}
+  update-homebrew-formula:
+    name: Run brew bump-formula-pr for iota on testnet releases
+    needs: release-build
+    runs-on: ubuntu-latest
+    # releasing iota cli on testnet releases because it lags `main` less than mainnet, but is more likely to be stable than devnet
+    if: ${{ contains( inputs.iota_branch, 'testnet') || contains( github.ref, 'testnet') }}
```

Comment:
This means it is a manual process?
For automatic testnet release triggers it should be checking for `${{ github.event_name == 'release' && contains(github.ref, '-rc')}}`